src/plugins/home/server/services/sample_data/sample_data_registry.ts

```diff
@@ -61,6 +73,10 @@ export class SampleDataRegistry {
           throw new Error(`Unable to register sample dataset spec because it's invalid. ${error}`);
         }
 
+        if (customIntegrations) {
+          registerSampleDataSetWithIntegration(customIntegrations, core, value);
```

Comment:
I know that `home` doesn't have great unit test coverage, but would be nice to have a test to confirm that we actually register the  data with the custom_integration plugin when calling `registerSampleDataset`

Reply:
(There's an API integration test confirming that the 3 default datasets are present in custom integrations, but would be good to be certain here that we are doing the job of adding new ones via `registerSampleDataset`)

Reply:
I added a test-suite `sample_data_registry.test.ts`, similar to `tutorial_registry.test.ts`.